model/src/commonMain/kotlin/FloorMap.kt

```diff
@@ -0,0 +1,18 @@
+package io.github.droidkaigi.confsched2019.model
+
+sealed class FloorMap {
```

Comment:
If you use an enum, I think you can write more simple 🙏 

```kotlin
  enum class FloorMap(val title: String) {
    FLOOR_1("1F"),
    FLOOR_5("5F")
  }
```

Reply:
Oh, I see. Thanks!

Reply:
Fixed it. 21d6c3e